sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlDialectSpecTest.java

```diff
@@ -3934,4 +3934,25 @@ public void testSimpleTableName() {
             Row.withSchema(singleField).addValues(15L).build());
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
+
+  @Test
+  public void testLogicalAndZetaSQL() {
+    String sql =
+        "SELECT LOGICAL_AND(x) AS logical_and"
+            + " FROM UNNEST(["
+            + true
```

Comment:
Nit: It's a little strange to interpolate Java booleans `.toString()` in a SQL query, can you just make them strings instead?

Reply:
resolved, thanks.